tests/integrational/fixtures/native_sync/objects_v2/channel_members/get_channel_members_with_pagination.yaml

```diff
@@ -0,0 +1,106 @@
+interactions:
+- request:
+    body: '{"set": [{"uuid": {"id": "test-fix-118-0"}}, {"uuid": {"id": "test-fix-118-1"}},
```

Comment:
from .yaml file, it seems related to `get` request. Then why I see `set` in body?

Reply:
To record this cassette i used `set_channel_members()` request with examples. This is helpful if you modify tests and want to re-record this .yaml file. In such case you just delete this file, change request and run test which stores all the communication in the .yaml. So in short: adding `set_channel_members()` to test saves time in future